package.json

```diff
@@ -2,16 +2,40 @@
   "name": "@eightfold.ai/octuple",
   "version": "2.51.1",
   "license": "MIT",
-  "main": "lib/octuple.js",
+  "sideEffects": [
+    "**/*.css"
+  ],
+  "exports": {
+    ".": {
+      "types": "./lib/octuple.d.ts",
+      "import": "./lib/octuple.mjs",
+      "require": "./lib/octuple.cjs"
+    },
+    "./lib/*": {
```

Comment:
Currently, the only other export is locale, so wouldn't "./lib/locale" suffice here? I don't recommend we export all from lib as it's not what we currently do.